contracts/ERC20.sol

```diff
@@ -3,9 +3,6 @@
 
 pragma solidity ^0.8.0;
 
-import "./../contracts/IERC20.sol";
-import "./../contracts/IERC20Metadata.sol";
-
```

Comment:
Minor, but I think including the interfaces just add unnecessary complexity here